bigquery/tests/integration.sh

```diff
@@ -6,7 +6,9 @@
 #   docker-compose up -d elasticsearch
 #   curl -XDELETE localhost:9200/1000_genomes && curl -XDELETE localhost:9200/1000_genomes_fields
 #   BILLING_PROJECT_ID=google.com:api-project-360728701457 docker-compose up --build -d indexer
-#   curl -s 'http://localhost:9200/1000_genomes/type/HG02924' | jq -rS '._source' > 'tests/integration_golden_index.json'
+#   curl -s 'http://localhost:9200/1000_genomes/type/HG02924' | jq -rS '._source' > 'tests/1000_genomes_golden.json'
+#   curl -s 'http://localhost:9200/1000_genomes/_mappings?pretty' | jq -rS '.' > 'tests/1000_genomes_mappings_golden.json'
+#   curl -s 'http://localhost:9200/1000_genomes_fields/_search' | jq -rS '.hits.hits' > 'tests/1000_genomes_fields_golden.json'
```

Comment:
This just gets 10 fields. Either change to get all fields, or add comment
```
# Store first 10 fields, which is good enough for the golden file.
```

Reply:
Changed to get all fields.